app/components/submission/SubmissionPage.js

```diff
@@ -30,7 +30,7 @@ const FormStep = ({ history, nextUrl, updateSubmission, ...props }) => (
 
 const SubmissionPage = ({ match, history }) => (
   <WithCurrentSubmission>
-    {(updateSubmission, initialValues) => (
+    {(initialValues, { updateSubmission, finishSubmission }) => (
```

Comment:
Why not go full [RORO](https://medium.freecodecamp.org/elegant-patterns-in-modern-javascript-roro-be01e7669cbd) on this 😄

Reply:
No particular reason. My thinking was that the second parameter is the mutations object. This would mean two things: first, the mutations are grouped together and second, you don't have to know all the possible mutations if you wrap your code in `<WithCurrentSubmission>`.

The blog post you linked makes sense, but I would normally have to think more about something before adopting it.